hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/common/OrderConverter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.restjava.common;
+
+import jakarta.inject.Named;
+import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.domain.Sort;
+
+@Named
+@ConfigurationPropertiesBinding
+public class OrderConverter implements Converter<String, Sort.Direction> {
+    @Override
+    public Sort.Direction convert(String order) {
+        return Sort.Direction.valueOf(order.toUpperCase());
```

Comment:
the static method `fromString` is case-insensitive. Also do we really need a custom converter? OK to keep the converter though.

```suggestion
        return Sort.Direction.fromString(order);
```